shared/src/main/scala/drt/shared/api/Arrival.scala

```diff
@@ -44,6 +44,24 @@ case class Arrival(Operator: Option[Operator],
     case Some(s) => s.suffix
   }
 
+  def displayStatus: ArrivalStatus = {
+
+    val fifteenMinutes = 15 * 60 * 1000
+
+    this.Estimated match {
+      case _ if this.ActualChox.nonEmpty => ArrivalStatus("On Chocks")
+      case _ if this.Actual.nonEmpty => ArrivalStatus("Landed")
+      case Some(e) if this.Scheduled + fifteenMinutes < e => ArrivalStatus("Delayed")
+      case Some(_) => ArrivalStatus("Expected")
+      case _ if this.Status.description.toLowerCase  == "redirected" | this.Status.description.toLowerCase  == "diverted" => ArrivalStatus("Diverted")
```

Comment:
it would be nice to have an `isDivertedStatus` method, like the `isCancelledStatus` one